Source/WebKit/Shared/Extensions/WebExtensionContextParameters.h

```diff
@@ -42,6 +42,8 @@ struct WebExtensionContextParameters {
     URL baseURL;
     String uniqueIdentifier;
 
+    HashMap<String, WallTime> grantedPermissions;
```

Comment:
nit: cold this use `WebExtensionContext::PermissionsMap`?

Reply:
Not really, stuff in Shared should be standalone.